.flask_version/run.py

```diff
@@ -0,0 +1,22 @@
+# This is a quick and dirty code!
+from flask import Flask
+from flask import request
+from sample_code.endpoints import endpoint
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+
+
+@app.route("/api/v1/test/entry", methods=["POST"])
+def test_entry():
+    content = request.get_json()
+    # to be fully equivalent we should manage mandatory params etc ... like this
```

Comment:
doesn't make sense without context Remove this comment

Reply:
removed